### PR TITLE
added test for git sources in sources.py

### PR DESCRIPTION
**Pull request by fazerlicourice7**

### CODE REVIEW

File: snapcraft/tests/test_sources.py [hunk 1]
```diff
@@ -325,3 +325,22 @@ def test_get_tar_source_from_uri(self):
...
+        for source in sources:
+            with self.subTest(key=source):
+
+                options = tests.MockOptions(source=source, source_type='git')
+                snapcraft.sources.get(
+                    sourcedir='dummy',
+                    builddir='dummy',
+                    options=options)
+
+                mock_pull.assert_called_once_with()
```
Comment (sergiusens) [inline snapcraft/tests/test_sources.py]: called once with what?

**Comment by sergiusens** [comment]
Looks good, just fix that assertion in the code.

After fixing, do this:

```
git add snapcraft/tests/test_sources.py 
git commit --ammend
git push [origin|fazer] --force
```

When ammending add this at the end in one line only:
```
LP: #XXXXX
```

Where `XXXXX` is the launchpad bug number

`origin` or `fazer` or whatever your remote is (however you originally ran `git push [something]`)

**Comment by fazerlicourice7** [comment]
It isn't called with anything: https://github.com/ubuntu-core/snapcraft/blob/master/snapcraft/sources.py#L271

**Comment by fazerlicourice7** [comment]
sergiusens, don't you mean git commit --edit ?

**Comment by sergiusens** [comment]
@fazerlicourice7 I actually meant `amend`, but it was considering you'd fix the assertion :wink:

**Comment by fazerlicourice7** [comment]
sergiusens, But there isn't anything to fix. That method 'pull()' isn't called with any parameters.

**Comment by sergiusens** [comment]
On Thu, Jan 14, 2016 at 11:12 PM, Fazer <notifications@github.com> wrote:

> sergiusens, But there isn't anything to fix. That method 'pull()' isn't
> called with any parameters.
>
Ah, then it is all good. I'm :+1: but @elopio has the final say.

**Comment by elopio** [comment]
Awesome. Take a look now that we hit the line we were missing:
https://coveralls.io/files/1827079269#L298

**Comment by sergiusens** [comment]
Just wanted to come back to this PR and say thank you!

**Comment by fazerlicourice7** [comment]
@elopio can you approve the task on GCI please?

**Pull request merged**